frontend/src/utils/time.ts

```diff
@@ -161,7 +178,7 @@ export function dateFnsFormat(
   return computed(() => {
     const i18n = usei18n();
 
-    args[args.length - 1] = merge(args, {
+    args[2] = merge(args[2], {
```

Comment:
Why? What if datefns change the amount of params this function has?

Reply:
Cause this doesn't work. Calling this method with `dateFnsFormat(new Date(item.PremiereDate), 'PPP')` as done in the persons page, `args[args.length-1]` just gives you the `PPP` argument. Hence, without my fix, it just erased that parameter.

Also, the merge logic was wrong. It was:

```typescript
args[args.length - 1] = merge(args, {
      locale: getDateFnsLocale(i18n.locale.value)
    });
```

So, it merged the locale into `[date, format]`. I don't remember the details, but it made an infinite object.